Lib/idlelib/editor.py

```diff
@@ -904,11 +905,12 @@ def RemoveKeybindings(self):
         # Called from configdialog.deactivate_current_config.
         self.mainmenu.default_keydefs = keydefs = idleConf.GetCurrentKeySet()
         for event, keylist in keydefs.items():
-            self.text.event_delete(event, *keylist)
+            with contextlib.suppress(ValueError):
+                self.text.event_delete(event, *keylist)
```

Comment:
I do not recall immediately what the issue was, I can look into that.

Reply:
Looking into it more, this particular suppress is useless, there is no errors. The following one does have errors though, erroring whenever it attempts to remove an event defined from an extension that was loaded from user configuration files.
```console
Partial Traceback:
    editwin.text.event_delete(event, *keylist)
  File "/usr/lib/python3.12/idlelib/multicall.py", line 391, in event_delete
    self.__binders[triplet[1]].unbind(triplet, func)
  File "/usr/lib/python3.12/idlelib/multicall.py", line 234, in unbind
    doit()
  File "/usr/lib/python3.12/idlelib/multicall.py", line 232, in <lambda>
    doit = lambda: self.bindedfuncs[triplet[2]][triplet[0]].remove(func)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: list.remove(x): x not in list
```
I will look deeper into this.

Reply:
Finished, turns out this error will never happen either, I am only having this error because my extension that monkeypatches IDLE is loaded after the builtin extensions have loaded, which leads to some issues later. This issue should never happen in a copy of IDLE that loads user extensions from the get go.

Removed suppression in 024095903e